extension/js/common/composer/composer-contacts.ts

```diff
@@ -38,13 +38,51 @@ export class ComposerContacts extends ComposerComponent {
   }
 
   initActions(): void {
-    this.composer.S.cached('input_to').on('keyup', Ui.event.prevent('veryslowspree', () => this.searchContacts()));
+    let preventSearchContacts = false;
+    this.composer.S.cached('input_to').on('keyup', Ui.event.prevent('veryslowspree', () => {
+      if (!preventSearchContacts) {
+        this.searchContacts();
+      }
+    }));
     this.composer.S.cached('input_to').on('keydown blur', Ui.event.handle(async (target, e) => {
```

Comment:
Please move this whole function body into a private method on the class, eg `recipientInputKeydownHandler`, with a comment that it does keyboard navigation in search results etc.

Then you could still leave the `blur` handler directly in here, that one is not too long.

I have a thing for functions/methods that can ideally fit on my screen vertically :)

Reply:
Done :+1: